### ISSUE TITLE
Date in site footer is broken

### ISSUE THREAD

**Issue by stlhood** [mozilla/community-portal#774]
The site footer includes the following text: 

> Portions of this content are copyright 1998-%s by individual mozilla.org contributors. 

I am guessing that "%s" is supposed to be a variable substitution for the current year?

![image](https://user-images.githubusercontent.com/42821/228365831-7dde904a-12d7-47c4-9b27-834fa9e6262f.png)

**Comment by couci** [comment]
@Mte90 can you take a look ?

**Comment by Mte90** [comment]
yes it is something easy to do but I am busy with other things in this period
